src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -71,13 +73,14 @@ import Pact.Types.Term (DefName(..), ModuleName(..))
 
 -- internal Chainweb modules
 
+import Chainweb.BlockHash
 import Chainweb.Pact.Service.Types (internalError)
-import Chainweb.Pact.Types (GasSupply(..), MinerInfo(..))
+import Chainweb.Pact.Types (GasSupply(..), MinerInfo(..), GasId(..))
 import Chainweb.Transaction (gasLimitOf, gasPriceOf)
+import Chainweb.Utils (sshow)
 
-------------------------------------------------------------------------------
+-- ------------------------------------------------------------------------ --
```

Comment:
weird spaces?

Reply:
Discussed - this is a convention Lars set, and I was conforming to. I'm going to go ahead and remove.